README.md

```diff
@@ -81,11 +105,14 @@ $ terraform init # not needed when override is in use
 
 Updating docs
 -------------
-Don't forget to add docs and examples to support your contribution. Update tfplugion docs when needed.
+Don't forget to add docs and examples to support your contribution. Update [tfplugindocs](//github.com/hashicorp/terraform-plugin-docs/releases/) when needed.
 ```sh
 $ tfplugindocs
 $ git add .
 ```
 
+### Contributing
```

Comment:
this is not a full guide, should we confuse contributors then ?

Reply:
removed